collectors/python.d.plugin/memcachier/memcachier.chart.py

```diff
@@ -0,0 +1,204 @@
+# Author: José Canal (joseliber)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+
+from bases.FrameworkServices.UrlService import UrlService
+
+
+ORDER = [
+    'cache',
+    'net',
+    'connections',
+    'items',
+    'evicted',
+    'expired',
+    'get',
+    'get_rate',
+    'set_rate',
+    'flush_rate',
+    'delete_rate',
+    'cas',
+    'delete',
+    'increment',
+    'decrement',
+    'touch',
+    'touch_rate',
+]
+
+CHARTS = {
+    'cache': {
+        'options': [None, 'Cache Size', 'MiB', 'cache', 'memcachier.cache', 'stacked'],
+        'lines': [
+            ['avail', 'available', 'absolute', 1, 1 << 20],
+            ['used', 'used', 'absolute', 1, 1 << 20]
+        ]
+    },
+    'net': {
+        'options': [None, 'Network', 'kilobits/s', 'network', 'memcachier.net', 'area'],
+        'lines': [
+            ['bytes_read', 'in', 'incremental', 8, 1000],
+            ['bytes_written', 'out', 'incremental', -8, 1000],
+        ]
+    },
+    'connections': {
+        'options': [None, 'Connections', 'connections/s', 'connections', 'memcachier.connections', 'line'],
+        'lines': [
+            ['curr_connections', 'current', 'incremental'],
+            ['total_connections', 'total', 'incremental']
+        ]
+    },
+    'items': {
+        'options': [None, 'Items', 'items', 'items', 'memcachier.items', 'line'],
+        'lines': [
+            ['curr_items', 'current', 'absolute'],
+            ['total_items', 'total', 'absolute']
+        ]
+    },
+    'evicted': {
+        'options': [None, 'Items', 'items', 'items', 'memcachier.evicted', 'line'],
+        'lines': [
+            ['evictions', 'evicted', 'absolute']
+        ]
+    },
+    'expired': {
+        'options': [None, 'Items', 'items', 'items', 'memcachier.expired', 'line'],
+        'lines': [
+            ['expired', 'expired', 'absolute']
+        ]
+    },
+    'get': {
+        'options': [None, 'Requests', 'requests', 'get ops', 'memcachier.get', 'stacked'],
+        'lines': [
+            ['get_hits', 'hits', 'percent-of-absolute-row'],
+            ['get_misses', 'misses', 'percent-of-absolute-row']
+        ]
+    },
+    'get_rate': {
+        'options': [None, 'Rate', 'requests/s', 'get ops', 'memcachier.get_rate', 'line'],
+        'lines': [
+            ['cmd_get', 'rate', 'incremental']
+        ]
+    },
+    'set_rate': {
+        'options': [None, 'Rate', 'requests/s', 'set ops', 'memcachier.set_rate', 'line'],
+        'lines': [
+            ['cmd_set', 'rate', 'incremental']
+        ]
+    },
+    'flush_rate': {
+        'options': [None, 'Rate', 'requests/s', 'flush ops', 'memcachier.flush_rate', 'line'],
+        'lines': [
+            ['cmd_flush', 'rate', 'incremental']
+        ]
+    },
+    'delete_rate': {
+        'options': [None, 'Rate', 'requests/s', 'delete ops', 'memcachier.delete_rate', 'line'],
+        'lines': [
+            ['cmd_delete', 'rate', 'incremental']
```

Comment:
Do we really need a separate chart per cmd operation? Why not put them in one chart?